salt/loader.py

```diff
@@ -441,6 +441,7 @@ def roster(opts, whitelist=None):
         opts,
         tag='roster',
         whitelist=whitelist,
+        pack={'__runner__': runner(opts)})
```

Comment:
Should I load it in the salt.roster.Roster class or in salt.client.ssh.SSH class?